src/main/java/Duke/gui/MainWindow.java

```diff
@@ -0,0 +1,74 @@
+package duke.gui;
+
+import duke.Duke;
+import duke.Pair;
+import javafx.fxml.FXML;
```

Comment:
I'm really nitpicking here but perhaps adding a line break between `import duke.Pair` and `import javafx.fxml.FXML` would adhere better to the coding standards? I have notice this in several other places too, maybe something to consider?
```suggestion
import duke.Pair;

import javafx.fxml.FXML;
```